src/common/reconciler_test.go

```diff
@@ -81,6 +54,11 @@ func TestReconcilerReconcile(t *testing.T) {
 			registration: testRegistration,
 			reconciler: common.NewServiceReconciler(&common.OpslevelClient{
 				GetServiceHandler: func(alias string) (*opslevel.Service, error) {
+					// TODO: this global var is a hack to get a service to be returned only once
```

Comment:
Original test does not check for the case where there are more than 1 matching aliases - I added this. 

Also ensured better code path checking - always set up handlers for at least the get, create, update operations. Ideally we should add them for all but the diff would be too big for current purpose